src/frontend/context/MigreringContext.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+
+import createUseContext from 'constate';
+
+import { useHttp } from '@navikt/familie-http';
+import { RessursStatus } from '@navikt/familie-typer';
+
+import { IInfotrygdSak, IInfotrygdsaker } from '../typer/infotrygd';
+import { Adressebeskyttelsegradering } from '../typer/person';
+
+const [MigreringProvider, useMigrering] = createUseContext(() => {
+    const [infotrygdsaker, settInfotrygdsaker] = React.useState<IInfotrygdSak[]>([]);
+
+    const { request } = useHttp();
+
+    const hentSakerForBruker = async (ident: string) => {
+        const hentetData = await request<{ ident: string }, IInfotrygdsaker>({
```

Comment:
Ønsker vi ikke spinner, feilmeldinger osv på siden, siden dere bruker await?

Reply:
Her må vi nok ta en prat, for ting rundt dette er nok litt ullent.